app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -87,7 +87,7 @@ def index
           repos = scoped_search(*base_args, options)
           csv_response(repos,
                        [:id, :name, :description, :label, :content_type, :arch, :url, :major, :minor,
-                        :cp_label, :content_label, :pulp_id, :container_repository_name,
+                        :content_label, :pulp_id, :container_repository_name,
```

Comment:
CSV column headers below this needs removal too, "Candlepin Label" is still there

Reply:
good catch!